sdk/js/src/Client.ts

```diff
@@ -469,20 +469,20 @@ export class DevCycleClient<
 
             this.onInitialized
```

Comment:
it's very weird we use `.then` / `.catch` here instead of `async` / `await`.

Reply:
yeah, agreed. I wanted to avoid changing it for the sake of minimizing risk, though I don't love having to mix .then with async/await